product_supplierinfo_for_customer/models/product_customerinfo.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# Copyright 2019 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import _, api, fields, models
+
+
+class ProductCustomerInfo(models.Model):
+    _inherit = "product.supplierinfo"
+    _name = "product.customerinfo"
+    _description = "Customer Pricelist"
+
+    name = fields.Many2one(string="Customer", help="Customer of this product")
```

Comment:
@hildickethan-S73 Comented yet this, but you don't change it. If you don't changes this, the module does not working. Please, change it and make a migration script to keep the data base information.
```suggestion
    parnter_id = fields.Many2one(string="Customer", help="Customer of this product")
```